docs/topics/db/sql.txt

```diff
@@ -54,6 +54,23 @@ Of course, this example isn't very exciting -- it's exactly the same as
 running ``Person.objects.all()``. However, ``raw()`` has a bunch of other
 options that make it very powerful.
 
+
+.. versionadded:: 2.1
+
+Likewise :class:`~django.db.models.query.QuerySet`, ``RawQuerySet`` also
+implements internal results caching (see :ref:`caching-and-querysets`).
+
+``RawQuerySet`` supports specifying prefetch related lookups. If prefetch
+related lookup is defined on :class:`~django.db.models.query.QuerySet`
+that will be carry forward to ``RawQuerySet``.
+
+.. warning::
+
+    ``RawQuerySet`` internal caching is not backward incompatible. If you
```

Comment:
Notes about backwards incompatibilities  go in the release notes under "Backwards incompatible changes in 2.1".